manifests/init.pp

```diff
@@ -22,7 +22,8 @@
   Stdlib::Filemode $rules_configdir_mode = $logrotate::params::rules_configdir_mode,
   String $root_user                      = $logrotate::params::root_user,
   String $root_group                     = $logrotate::params::root_group,
-  Array[String] $logrotate_args          = []
+  Array[String] $logrotate_args          = [],
```

Comment:
Can you please enforce a minimal string length like this:
```puppet
Array[String[1]]
```

Reply:
Yes, I agree that is better. However, it belongs to pull request 80. I will fix it there.
https://github.com/Heidistein/puppet-logrotate/tree/pr80fix

Reply:
Fixed in PR #127 
I added only a training comma here, this was unrelated to this change.